webapp/app.py

```diff
@@ -709,6 +710,8 @@ def takeovers_index():
 core_services_guide.init_app(app)
 
 
+app.add_url_rule("/user-country.json", view_func=get_user_country_by_ip)
```

Comment:
Currently, https://ubuntu-com-12298.demos.haus/user-country.json contains the header:

```
cache-control: max-age=60, stale-while-revalidate=86400, stale-if-error=300
```

Which means users will still see cached versions of this endpoint that were generated by other users.

You need to change cache-control headers on this endpoint so it doesn't get cached. I think ideally that should mean you should use `cache-control: private`, which would allow this resource to be cached by the user's own browser, which is slightly better for performance. `no-store` would also work, but would mean the user never gets to cache the resource. You could either do this in the function itself or [here](https://github.com/canonical/ubuntu.com/blob/main/webapp/app.py#L1057).